TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/common/sockets/BasicStringHandler.java

```diff
@@ -21,6 +30,28 @@ protected void channelRead0(ChannelHandlerContext ctx, String message) throws Ex
 
     public abstract void handleMessage(Message message, String originalMessage, Channel channel);
 
+    public List<MessageHandler> getMessageHandlers(MessageType messageType) {
+        List<MessageHandler> returnedMessageHandlers = new ArrayList<>();
+
+        for(MessageHandler messageHandler : messageHandlers) {
+            if(messageHandler.getMessageType().equals(messageType))
+                returnedMessageHandlers.add(messageHandler);
+        }
```

Comment:
Replace this with ` = messageHandlers.filter(handler -> handler.getMessageType().equals(messageType))`.

It would be even better to use a `HashMap<MessageType, Set<MessageHandler>>` so that we can simply use `map.get(messageType)`.